packaging/opensuse-42.2/snapd.spec

```diff
@@ -171,7 +173,7 @@ install -m 644 -D packaging/opensuse-42.2/permissions %buildroot/%{_sysconfdir}/
 install -m 644 -D packaging/opensuse-42.2/permissions.paranoid %buildroot/%{_sysconfdir}/permissions.d/snapd.paranoid
 # Install the systemd units
 make -C data/systemd install DESTDIR=%{buildroot} SYSTEMDSYSTEMUNITDIR=%{_unitdir}
-for s in snapd.autoimport.service snapd.system-shutdown.service; do
+for s in snapd.autoimport.service snapd.system-shutdown.service snap-repair.timer snap-repair.service; do
```

Comment:
I was under the impression that the repair functionality is only applicable to core systems. I would not recommend using it in any classic distribution where existing packaging systems are sufficient to recover from any issue.